### PR TITLE
fix(examples): fix errors in logger and metrics examples

### PR DESCRIPTION
**Pull request by michaelbrewer**
## Description of your changes

- Use shortened imports where possible
- Use 'serviceName' instead of 'service'
- Correct some of the relative path imports to work again
- Correct spacing around '}'
- Typo in 'metrics/examples/hello-world.ts'
- Correct link to Tracer examples from Readme

### How to verify this change

Run `npm test` and open and fix examples in the metrics and logger packages.

### Related issues, RFCs

- https://github.com/awslabs/aws-lambda-powertools-typescript/issues/510

### PR status

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

## Checklist

- [X] [My changes meet the tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
- [X] I have performed a *self-review* of my own code
- [X] I have *commented* my code where necessary, particularly in areas that should be flagged with a TODO, or hard-to-understand areas
- [X] I have made corresponding changes to the *documentation*
- [X] My changes generate *no new warnings*
- [X] The *code coverage* hasn't decreased
- [X] I have *added tests* that prove my change is effective and works
- [X] New and existing *unit tests pass* locally and in Github Actions
- [X] Any *dependent changes have been merged and published* in downstream module
- [X] The PR title follows the [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/.github/semantic.yml#L2)

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/metrics/examples/hello-world.ts [hunk 1]
```diff
@@ -17,6 +17,6 @@ const lambdaHandler = async (): Promise<void> => {
 };
 
 const handlerWithMiddleware = middy(lambdaHandler)
-  .use(logMetrics(metrics}));
```
Comment (michaelbrewer) [inline packages/metrics/examples/hello-world.ts:20]: This is an actual compile error in TS

**Comment by michaelbrewer** [comment]
@dreamorosi @flochaz - maybe in the long term, some of these examples will be dropped or added to the test harness or move to a separate repo :)

**Comment by flochaz** [comment]
Can you open an issue to explain first what is the problem ?

**Comment by flochaz** [comment]
And let's try to follow [contribution guidelines](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/CONTRIBUTING.md) and have PR fixing one thing issue at a time. thx !

**Comment by michaelbrewer** [comment]
Ok. These examples have types and errors in them. I can raise issues for it too. 

And the link for the tracer example is wrong again.

**Comment by michaelbrewer** [comment]
> Can you open an issue to explain first what is the problem ?

https://github.com/awslabs/aws-lambda-powertools-typescript/issues/510

**Comment by michaelbrewer** [comment]
@flochaz - there is no explicit call out for an issue before a PR? Also there is alot of typos and issues in this guide referencing python or CDK:

Example of CDK project references:

Screen Shot 2022-01-25 at 10 23 54 AM

Examples of Python references:

Screen Shot 2022-01-25 at 10 28 13 AM

**Review by flochaz — APPROVED** [review]

**Comment by flochaz** [comment]
Thanks a lot for your contribution and we will make sure we will be clearer in the contributing guide on our expectation and especially on utility of issue creation and PR scope.

**Review by saragerion — APPROVED** [review]
Thanks for this contribution!

**Pull request merged**